public/app/features/dashboard/containers/DashboardPage.tsx

```diff
@@ -307,19 +306,16 @@ export class UnthemedDashboardPage extends PureComponent<Props, State> {
 
     return (
       <div className={styles.dashboardContainer}>
-        {kioskMode !== KioskMode.Full && (
-          <div aria-label={selectors.pages.Dashboard.DashNav.nav}>
-            <DashNav
-              dashboard={dashboard}
-              title={dashboard.title}
-              folderTitle={dashboard.meta.folderTitle}
-              isFullscreen={!!viewPanel}
-              onAddPanel={this.onAddPanel}
-              kioskMode={kioskMode}
-              hideTimePicker={dashboard.timepicker.hidden}
-            />
-          </div>
-        )}
+        <div aria-label={selectors.pages.Dashboard.DashNav.nav}>
```

Comment:
Is this div needed outside DashNav can't this aria-label be moved inside the DashNav component?